src/gordon_janitor_gcp/plugins/authority.py

```diff
@@ -89,9 +89,9 @@ def _validate_config(self):
                    'Resource Manager.')
             logging.error(msg)
             raise exceptions.GCPConfigError(msg)
-        if not self.config.get('zone'):
-            msg = ('The ID of the target Cloud DNS managed zone is required to '
-                   'identify which zone records should belong to.')
+        if not self.config.get('dns_zone'):
+            msg = ('The name of a zone, i.e. "example.com.", is required to '
```

Comment:
Maybe, `'The FQDN of a zone, i.e. "example.com."...` ? (or "fully qualified domain name")

Reply:
It seems like it applies, but I've never used FQDN to refer to the name of anything other than a server before, so I'm little hesitant to use it for a zone, unless I get some confirmation it's valid usage.

Reply:
We could also use _absolute zone_, since this is the terminology used by the [rfc](https://tools.ietf.org/html/rfc1035) (_Domain names that end in a dot are called absolute, and are taken as complete._) just because the dot is important and it seems easy to miss. Wikipedia says that FQDN and absolute domain (zone?) are interchangeable.

Reply:
I actually just looked up some providers and it looks like "absolute zone" is not as widespread as I thought, so unless everybody complains because they really loved it, I'm probably going to remove the '.' code and messaging.

NS1: https://ns1.com/api#create-a-new-dns-zone

DYN (note you have to check the example request under SOAP to see an example):
https://help.dyn.com/create-primary-zone-api/

Even Route53 seems to have the trailing '.' as optional: https://docs.aws.amazon.com/Route53/latest/APIReference/API_CreateHostedZone.html#API_CreateHostedZone_RequestSyntax